src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,145 @@
+          <input
+            type="email"
+            name="email"
+            placeholder="Email"
+            {...register('email')}
+            className="sm:w-96 w-80 shadow-lg text-primary focus:outline-none focus:tertiary focus:ring-1 focus:ring-tertiary rounded-md placeholder:italic placeholder:text-tertiary px-3 py-1 mt-1 block duration-500"
+          />
+          <p className="text-red-800 font-semibold">
+            {errors?.email?.message}
+          </p>
+        </label>
+        <label htmlFor="password" className="mt-3">
+          <span className="text-primary font-semibold">Password</span>
+          <input
+            type="password"
+            name="password"
+            placeholder="Password"
+            {...register('password')}
+            className="sm:w-96 w-80 shadow-lg focus:outline-none focus:tertiary focus:ring-1 focus:ring-tertiary text-primary rounded-md placeholder:italic placeholder:text-tertiary px-3 py-1 mt-1 block duration-500"
+          />
+          <p className="text-red-800 font-semibold">
+            {errors?.password?.message}
+          </p>
+        </label>
+        <label htmlFor="confirmPassword" className="mt-3">
+          <span className="text-primary font-semibold">Confirm Password</span>
+          <input
+            type="password"
+            name="confirmPassword"
+            placeholder="Confirm Password"
+            {...register('confirmPassword')}
+            className="sm:w-96 w-80 shadow-lg focus:outline-none focus:tertiary focus:ring-1 focus:ring-tertiary text-primary rounded-md placeholder:italic placeholder:text-tertiary px-3 py-1 mt-1 block duration-500"
+          />
+          <p className="text-red-800 font-semibold mt-1 mb-3">
+            {errors?.confirmPassword?.message && 'Passwords should match'}
+          </p>
+        </label>
+        <button
+          type="submit"
+          className="sm:w-96 w-80 font-semibold shadow-lg my-2 bg-primary text-background py-2 rounded-md hover:bg-tertiary hover:text-primary duration-500"
+        >
+          Sign Up
+        </button>
+        <p className="text-lg text-primary self-center mt-2">
+          Already Have an Account?
+          <Link
+            to="/signin"
+            className="underline hover:text-secondary duration-300"
+          >
+            Sign In
+          </Link>
+        </p>
+        <p className="text-lg text-primary font-bold self-center my-4">
+          OR
+        </p>
+        <p className="text-xl text-primary font-semibold self-center mb-6">
+          Sign Up With
+          <button type="button">
+            <BsFacebook className="inline pb-1 h-9 w-9 hover:text-secondary mx-1 duration-200" />
+            or
+          </button>
+          <button type="button">
+            <BsGoogle className="inline pb-1 h-9 w-9 hover:text-secondary mx-1 duration-200" />
+            .
+          </button>
+        </p>
+      </form>
+    </div>
+  );
+}
+
+export default SignUp;
```

Comment:
	Make some spacing between Sign in button and inputs the same as sign up button
	Also move Sign in with Google and Facebook to the top of the form
	There is no “forgot your password” option.
	There is no “remember me” option if the user prefers to stay logged in

Reply:
 If you can make "hr" element along with “OR” and make it small letters.
 It’s highly recommended to use official logos for Google and Facebook.
 If you increase the padding for input fields.
 Make some spaces for “Already Have an Account?Sign In” and make the font bolder if possible.